config/.template-lintrc.js

```diff
@@ -17,6 +17,7 @@ module.exports = {
     'style-concatenation': true,
     'triple-curlies': true,
     'unused-block-params': false,
+    'simple-unless': false,
```

Comment:
Can you elaborate why we disabling this rule?

Reply:
@seban this is a new rule that is currently throwing us a 7 warnings in the webapp 
`Using an {{else}} block with {{unless}} should be avoided`
and does not bring any special value. I spoken with @cinkonaap about this, we have decided that at the moment it is possible to disable it.